operators/assisted-service-operator/0.0.7/assisted-service-operator.clusterserviceversion.yaml

```diff
@@ -47,7 +47,7 @@ metadata:
     containerImage: quay.io/ocpmetal/assisted-service@sha256:82b37b50fdd2a8880ee5f519cbd33be24752fa70e527e25662c308f917e7ddf2
     createdAt: 2021-07-08T14:02:34Z
     description: The Infrastructure Operator for Red Hat OpenShift is responsible for managing the deployment of the Assisted Service.
-    olm.skipRange: '>=0.0.0 <0.0.7'
```

Comment:
This project has not a bundle version 0.0.0. Then, it fails when we try to manipulated the bundle to add in the catalogs

Reply:
@flaper87,@djzager,@rwsu

Reply:
@djzager is the expert with olm and the current versioning strategy. As far as I can tell, this looks good to me.